games-emulation/mgba/mgba-0.10.5.ebuild

```diff
@@ -0,0 +1,131 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LUA_COMPAT=( lua5-{3..4} )
+inherit cmake lua-single xdg
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mgba-emu/mgba.git"
+else
+	SRC_URI="https://github.com/mgba-emu/mgba/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+fi
+
+DESCRIPTION="Game Boy Advance Emulator"
+HOMEPAGE="https://mgba.io/"
+
+LICENSE="MPL-2.0 BSD LGPL-2.1+ public-domain discord? ( MIT )"
+SLOT="0/$(ver_cut 1-2)"
+IUSE="
+	debug discord elf ffmpeg gles2 gles3 gui libretro
+	lua +opengl +sdl +sqlite test
+"
+REQUIRED_USE="
+	gui? ( || ( gles2 gles3 opengl ) sqlite )
+	lua? ( ${LUA_REQUIRED_USE} )
+"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	media-libs/libpng:=
+	sys-libs/zlib:=[minizip]
+	debug? ( dev-libs/libedit )
+	elf? ( dev-libs/elfutils )
+	ffmpeg? ( media-video/ffmpeg:= )
+	gles2? ( media-libs/libglvnd )
+	gles3? ( media-libs/libglvnd )
+	lua? (
+		${LUA_DEPS}
+		dev-libs/json-c:=
+	)
+	opengl? ( media-libs/libglvnd )
+	gui? (
+		dev-qt/qtbase:6[gui,network,opengl,widgets]
```

Comment:
I take it you've copied the 9999 ebuild, but that ebuild is meant for the 0.11.x branch while this is a maintenance release for 0.10.x -- it notably still uses Qt5 rather than Qt6.

Should base this bump on the 0.10.4 ebuild instead, glancing at the tarballs I don't think anything need changing from it except for the cmake-4 stuff.

Reply:
Oh, you're right. Sorry, didn't notice that.